src/pljson_parser.impl.sql

```diff
@@ -22,24 +22,29 @@ create or replace package body pljson_parser as
   */
 
   decimalpoint varchar2(1 char) := '.';
+  -- 400 - random value between 200 - 500
```

Comment:
In order for this comment to be useful, it must explain _why_ the `400` value was chosen. There must be a technical reason for the choice.